src/NuGet.Services.AzureSearch/SearchService/AzureSearchService.cs

```diff
@@ -34,6 +35,15 @@ public async Task<V2SearchResponse> V2SearchAsync(V2SearchRequest request)
         {
             if (request.IgnoreFilter)
             {
+                if (request.PackageType != null && request.PackageType.Any())
+                {
+                    throw new InvalidOperationException($"Can't apply the packageType filter on the Hijack index");
```

Comment:
nit: don't need `$`
This should bubble out at an HTTP 400, right? Does it?

Reply:
You should be able to add a functional test to verify this does return an HTTP 400 response.